CMakeLists.txt

```diff
@@ -335,6 +335,11 @@ if(BUILD_TESTS)
       historical_queries_test PRIVATE http_parser.host sss.host
     )
 
+    add_unit_test(
+      indexing_test ${CMAKE_CURRENT_SOURCE_DIR}/src/indexing/test/indexing.cpp
+    )
+    target_link_libraries(indexing_test PRIVATE sss.host)
```

Comment:
Is there an unforeseen reason why the indexing test needs sss?

Reply:
Yes. To rekey, we create a `ccf::ShareManager`:
```
#include "node/share_manager.h"
->
`#include "secret_share.h"`
->
in function 'ccf::SecretSharing::split(...)' : undefined reference to 'sss_create_shares'
```

We could break this chain by moving some implementation from headers, but for now we have an unexpected `sss` dependency in several unit tests.